js/models/conversations.js

```diff
@@ -1616,12 +1621,12 @@
       await this.updateProfile();
     },
     async setProfile(profile) {
-      if (_.isEqual(this.get('profile'), profile)) return;
-
-      this.set({ profile });
-      await window.Signal.Data.updateConversation(this.id, this.attributes, {
-        Conversation: Whisper.Conversation,
-      });
+      if (!_.isEqual(this.get('profile'), profile)) {
```

Comment:
Any reason you changed this?
In general, I usually give preference to whatever:
- reduces the diff
- reduces the indentation

Reply:
Ok, didn't catch the first time that now `updateProfile()` is called regardless.
Thanks.